server/etcdserver/api/etcdhttp/health.go

```diff
@@ -23,11 +23,12 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"go.uber.org/zap"
 
+	"go.etcd.io/raft/v3"
```

Comment:
the goimport check is not reliable. The `go.etcd.io` should be in one group. 

https://github.com/etcd-io/etcd/blob/185ddb2e78a6f856c6151466d57ea6306ff17c57/tools/.golangci.yaml#L29

Reply:
Thanks! @fuweid 

It always bears me... After etcd and raft splits, Goland does not recognize raft as current project packages... 

FWIW, I have updated the code style perference and it is now align with the `.golangci` configuration!

<img width="980" alt="image" src="https://github.com/etcd-io/etcd/assets/54131596/c712505a-b9c9-4af7-b0a5-580162d300ef">